app/views/protocols/index.html.erb

```diff
@@ -18,84 +18,18 @@
-            <span class="hidden-xs">&nbsp;<%= t("protocols.index.publish") %></span>
-          </button>
-        <% end %>
-      </div>
-      <div id="import-export-protocols" class="sci-btn-group" role="group">
-        <button class="btn btn-light btn-open-file <%= 'disabled hidden' unless can_create_protocols_in_repository?(@current_team) %>"
-              data-toggle="dropdown"
-              aria-haspopup="true"
-              aria-expanded="false">
-          <span class="fas fa-download"></span><span class="hidden-xs">&nbsp;<%= t("protocols.index.import") %></span>
-        </button>
-        <% if can_create_protocols_in_repository?(@current_team) %>
-          <ul class="dropdown-menu">
-            <li>
-              <a class="btn-link-alt btn-default-link btn-open-file" <%= can_create_protocols_in_repository?(@current_team) ? 'data-action="import"' : 'disabled="disabled"' %>>
-                <span class="fas fa-paperclip"></span>
-                <span class="hidden-xs"><%= t("protocols.index.import_alt") %></span>
-                <input type="file" value="" accept=".eln" data-role="import-file-input"
-                data-team-id="<%= @current_team.id %>"
-                data-type="<%= @type %>" data-import-url="<%= import_protocols_path %>"
-                <%= 'disabled="disabled"' unless can_create_protocols_in_repository?(@current_team) %>>
-              </a>
-            </li>
-            <li>
-            <%= link_to "#modal-import-json-protocol", data: { toggle: 'modal' } do %>
-              <span class="fas fa-paperclip"></span>
-              <span class="hidden-xs"><%= t("protocols.index.import_json") %></span>
-            <% end %>
-            </li>
-          </ul>
-        <% end %>
-        <button class="btn btn-light disabled hidden" data-action="export" data-export-url="<%= export_protocols_path() %>">
-          <span class="fas fa-upload"></span>
-          <span class="hidden-xs">&nbsp;<%= t("protocols.index.export") %></span>
-        </button>
-      </div>
-
-      <div class="sci-btn-group" role="group">
-        <button class="btn btn-light disabled hidden" data-action="archive" data-url="<%= archive_protocols_path %>">
-          <span class="fas fa-archive"></span>
-          <span class="hidden-xs">&nbsp;<%= t("protocols.index.archive_action") %></span>
-        </button>
-      </div>
-      <div class="protocols-description">
-        <%= t("protocols.index.archive.description") %>
-      </div>
-      <div class="sci-btn-group" role="group">
-        <button class="btn btn-light disabled hidden" data-action="restore" data-url="<%= restore_protocols_path %>">
-          <span class="fas fa-sync-alt"></span>
-          <span class="hidden-xs">&nbsp;<%= t("protocols.index.restore") %></span>
-        </button>
-      </div>
-      -->
-      <%= render partial: "protocols/index/protocols_datatable.html.erb" %>
+      <% if @type == :external_protocols %>
```

Comment:
Maybe create a completely new page for external protocol? Because it's not called external protocols anymore